eng/common/scripts/Helpers/Metadata-Helpers.ps1

```diff
@@ -80,3 +80,54 @@ function GetPrimaryCodeOwner ([string]$TargetDirectory)
     Write-Warning "No code owner found in $TargetDirectory."
     return $null
 }
+
+function GetDocsMsService($clientPackageInfo, $mgmtPackageInfo, $serviceName) 
+{
+  $service = $serviceName.ToLower().Replace(' ', '').Replace('/', '-')
+  if ($clientPackageInfo -and $clientPackageInfo[0].MSDocService) {
+    # Use MSDocService in csv metadata to override the service directory
+    # TODO: Use taxonomy for service name -- https://github.com/Azure/azure-sdk-tools/issues/1442
```

Comment:
nit: reading this code, my understanding is that using the clientPackageInfo.MSDocService IS the taxonomy. So we can get rid of this doc comment right?

Reply:
Ideally we want to read the ms.service from some taxonomy tools/database. Docs team does not provide such tools yet.